pkgs/games/anki/bin.nix

```diff
@@ -46,9 +47,10 @@ in
 
 if stdenv.isLinux then buildFHSUserEnv (appimageTools.defaultFhsEnvArgs // {
   name = "anki";
+  targetPkgs = pkgs: (with pkgs; [ xorg.libxkbfile krb5 ]);
 
   runScript = writeShellScript "anki-wrapper.sh" ''
-    exec ${unpacked}/bin/anki
+    DISABLE_QT5_COMPAT=1 exec ${unpacked}/bin/anki
```

Comment:
Same here. What does this env var mean, why do we need it and when can we remove it?

Reply:
Anki worked without this environment variable, but printed out a warning to the console saying that it ran in QT5 compatibility mode, and that this environment variable needed to be set to not run it in compatibility mode.

Reply:
I think we might not want to override this behaviour. I don't use KDE but it being QT6 might not interface with the rest of the NixOS desktop which is still QT5 AFAIK.

Reply:
Addressed all of the issue comments and tried adding support for aarch64 as well, but again, I cannot test that since I don't have access to apple silicon to run anki on. Would be great if someone else could confirm those changes.